www/docs/src/components/hero/hero.tsx

```diff
@@ -0,0 +1,147 @@
+import { type Props } from '@astrojs/starlight/props'
+import CliInstall from '@/components/cli-install/cli-install'
+import { Button } from '@/components/ui/button'
+import { ArrowUpRight } from 'lucide-react'
+
+const navigateTo = (
+  e: React.MouseEvent<HTMLButtonElement>,
+  url: string,
+  external?: boolean,
+) => {
+  e.preventDefault()
+  if (external) {
+    window.open(url, '_blank')
+  } else {
+    window.location.href = url
+  }
+}
```

Comment:
We should try to use `<a>` tags along with `target="_blank"` for external links so they work without JS enabled and so we don't have to remember to configure behavior for cltr click, etc.